webpack/src/js/main.js

```diff
@@ -0,0 +1,217 @@
+/*

```

Comment:
Das JS ist ein wenig Out of Date!

Reply:
https://github.com/ToolboxBodensee/toolbox-webseite/blob/master/assets/js/main.js

Reply:
oh scheine ich beim rebasen nicht aktualisiert zu haben

Reply:
ja, kann man ja alles noch nachholen :wink:

Reply:
ja klar gut dass du es gesehen hast ;)